### ISSUE TITLE
Import IDLs

### ISSUE THREAD

**Issue by dahifi** [labeleven-dev/bettercallsol#82]
You have a list of program IDs that you've preloaded. Where possible, import the IDLs from anchor registry, or allow JSON import, allowing a drop down for each instruction, which will populate the account structs for the instructions and serialize the parameter data. 

See Goki's transaction builder for an implementation of this: https://devnet.goki.so/wallets/5NwgCo9qxfhLxaWbLgxRvuG1HcYnimPtezuR9xiqeuZX/propose

**Comment by sohrab-** [comment]
This can be done today already: https://twitter.com/sohrabwashere/status/1559253728024154112

And there are a couple of items on the roadmap as well: https://github.com/labeleven-dev/bettercallsol/issues/23 and adding a library of well-known IDLs. 

Do you see any other gaps we should cover?

**Comment by dahifi** [comment]
No, this is great. I missed the import functionality in the hamburger when I was checking this out yesterday. The seed generation feature is fantastic as well. 

Might be nice to allow users to set constants or do some RPC calls. Losing your data on refresh is frustrating, maybe a file browser to save txs. ReplIt for Solana?

**Comment by sohrab-** [comment]
Yeah, there is a good chunk of features that still need to be added.

* I have this one on the roadmap that should help with constants https://github.com/labeleven-dev/bettercallsol/issues/14
* Originally had the user session persist across page refresh but that made it impossible to use BCS in multiple tabs concurrently. I just created this one for a more sophisticated solution https://github.com/labeleven-dev/bettercallsol/issues/85.
